src/App.vue

```diff
@@ -380,6 +384,10 @@ export default {
 				this.$store.dispatch('hideSidebar')
 			}
 		},
+		// Upon pressing ctrl+f, focus the search box in the left sidebar
+		handleAppSearch() {
+			document.querySelector('.conversations-search')[0].focus()
```

Comment:
And now add soemthing that when it is already focused default ctrl+f happens?

Reply:
It already works, the second ctrl+f triggers the browser search